airflow/dag_processing/bundles/git.py

```diff
@@ -40,20 +109,35 @@ class GitDagBundle(BaseDagBundle):
     :param repo_url: URL of the git repository
     :param tracking_ref: Branch or tag for this DAG bundle
     :param subdir: Subdirectory within the repository where the DAGs are stored (Optional)
+    :param git_conn_id: Connection ID for SSH connection to the repository (Optional)
```

Comment:
```suggestion
    :param git_conn_id: Connection ID for SSH/token based connection to the repository (Optional)
```

This support auth tokens too right?